tests/integration/pilot/tunneling/gateway/tcp/virtual-service.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: external-app-tunneling-tcp
+spec:
+  hosts:
+  - external-app.testdomain
+  gateways:
+  - istio-egressgateway
+  - mesh
```

Comment:
nit: I would split these up. I think its a really bad practice to couple them, even though that is what our docs and some tests do. The more tests we add that do it the more likely users will copy it.